config/locales/en.yml

```
@@ -1362,6 +1362,7 @@ en:
     profile: Profile
     relationships: Follows and followers
     statuses_cleanup: Automated post deletion
+    strikes: Moderation strikes
```

Would this be more understandable?

```suggestion
    strikes: Strikes against your account
```

---

maybe just "Moderation actions"? "Strikes" feels a little harsh.

---

They're called [strikes on YouTube](https://support.google.com/youtube/answer/2802032?hl=en) so people would be familiar with the concept